pegasus/migrations/109_add_hoc_activity_source.rb

```diff
@@ -0,0 +1,11 @@
+Sequel.migration do
+  up do
+    add_column :hoc_activity, :source, String, size: 50
+    add_index :hoc_activity, :source
+  end
+
+  down do
+    drop_index :hoc_activity, :source
+    drop_column :hoc_activity, :source
+  end
+end
```

Comment:
We recently added a new indexed column to `users` (17M rows) which took about half an hour; `hoc_activities` has ~210M rows, so I'd guesstimate this would take about six hours. Have we run migrations that take that long before?